src/foam/core/Interface.js

```diff
@@ -61,6 +61,12 @@ foam.CLASS({
       label: 'Generate Proxy',
       help: 'If enabled, causes automatic proxy generation.'
     },
+    {
+      class: 'Boolean',
+      name: 'null',
```

Comment:
This seems like a poor choice of name and likely to conflict with JS reserved words. Maybe 'nullStrategy' or something else.